rivery_cli/cli/activities.py

```diff
@@ -83,13 +86,35 @@ def download_run_logs(ctx, **kwargs):
     click.echo(
         f'Run ID {run_id} logs fetched successfully.'
     )
+
+    # Making the data prettier
+    # Removing the last char (empty line)
+    data = str(resp.content).replace("\\r\\n", "\r\n")[:-1]
```

Comment:
you can read this by read the csv as stream. Better than replace data as it is.

Reply:
No need to write the data into a file, and this can be also problematic in cases we don't have permission to do that. What we can do is push the data into StringIO or BytesIO, and then provide the csv reader the StringIO object to read from. After that, you can read it by prettytable. :)

Reply:
BTW - Make it optional. Means, the user can make the descision to output the data into files/directly to the console.